src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/NumberSelection/NumberSelection.styles.ts

```diff
@@ -1,8 +1,7 @@
 import { styled } from '@mui/material';
 
 import { StyledFlexColumn, StyledFlexTopCenter } from 'shared/styles/styledComponents';
```

Comment:
pls use import from 'shared/styles' also for these components

Reply:
done